modules/search_api_solr_autocomplete/src/Plugin/search_api_autocomplete/suggester/Spellcheck.php

```diff
@@ -16,7 +16,7 @@
  * work.
  *
  * @SearchApiAutocompleteSuggester(
- *   id = "search_api_solr_spellcheck",
+ *   id = "search_api_solr_autocomplete_spellcheck",
```

Comment:
Why changing the ID when the functionality is moved?

Reply:
@mkalkbrenner as we keep the old classes deprecated, how will work 2 plugin classes with the same ID?